sym-aws-go-approvals/internal/sym.go

```diff
@@ -0,0 +1,18 @@
+package internal
+
+import (
+	"fmt"
+
+	"github.com/symopsio/types/go/enums"
+	"github.com/symopsio/types/go/models"
+)
+
+// GetIdentity is a utility function to get a user identity for a given service
+func GetIdentity(user *models.User, service enums.Service) (*models.User_Identity, error) {
```

Comment:
maybe `FindIdentity`? I expected `GetIdentity` to do a fetch of some kind

Reply:
changed